sig-multicluster/annual-report-2022.md

```diff
@@ -4,66 +4,67 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Significant progress on KEP-1645, the [MCS API](https://sigs.k8s.io/mcs-api) now with conformance tests
+   - Developed the About API for ClusterSets
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - SIG-MC is building a website to track and communicate multicluster related initiatives, patterns, and best practices. ([contribute](https://github.com/kubernetes-sigs/sig-multicluster-site))
+   - We are working with the Gateway API project to formalize interaction with multi-cluster services
+   - The SIG is exploring cross-cluster workload management - what might it take to support multicluster controllers? and the [work API](https://github.com/kubernetes-sigs/work-api)
 
 
 
 3. KEP work in 2022 (v1.24, v1.25, v1.26):
   - alpha:
     - [2149 - ClusterID for ClusterSet Identification](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid) - v1.24
+  - progress to beta:
+      [1645 - MCS API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
 
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
+   - There has been continued and repeated interest in multicluster controller development, but we need more people to come and share specific use cases, previous attempts at implementations, and/or concerns before we can hope to propose any generally useful solutions.
```

Comment:
Is there a repo / issue that can be linked here? If someone external to the project looked at it, where would they go to provide those use-cases?

Reply:
Linked a brainstorming doc the community has put together